### ISSUE TITLE
Update provider to latest version 1.9.0 beta

### ISSUE THREAD

**Issue by alexvanderberkel** [siderolabs/terraform-provider-talos#214]
Would it be possible to update the provider to support the 1.9.0-beta.0 version? 

Thanks for all the work you're putting into Talos!

**Comment by smira** [comment]
Yes, the update is coming, but a release will only come with 1.9.0 final.

There is no support and changes might be coming while Talos is in beta.

**Comment by frezbo** [comment]
https://github.com/siderolabs/terraform-provider-talos/releases/tag/v0.7.0

**Comment by alexvanderberkel** [comment]
Thanks for the work you do!!
